src/python/grpcio_tests/tests/testing/_connection_test.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2017 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+import unittest
+
+from grpc.framework.foundation import logging_pool
+import grpc_testing
+
+from tests.testing import _application_testing_common
+from tests.testing import _client_application
+from tests.testing import _server_application
+
+
+class ConnectionTest(unittest.TestCase):
+
+    def setUp(self):
+        self._application_thread_pool = logging_pool.pool(1)
+        self._fake_time = grpc_testing.fake_time(time.time())
+        self._real_time = grpc_testing.real_time()
+        self._fake_time_connection = (grpc_testing.connection_from_descriptions(
+            {
```

Comment:
Although the `FirstServiceServicer` class (an instance of which is part of the system under test here) takes no construction parameters and could be a module-scope constant singleton within the Python interpreter; I'd like to retain construction here because I think it's important to exemplify constructing the entire system-under-test within the test's `setUp` method.

Not just important, but correct-with-respect-to-testing-orthodoxy too. :-)

Is that worth putting in a comment?